ogm/releases/5.3/index.adoc

```diff
@@ -0,0 +1,18 @@
+:awestruct-layout: project-releases-series
+:awestruct-project: ogm
+:awestruct-series_version: "5.3"
+
+=== Core
+
+==== Hibernate ORM 5.2 support
+
+As of Hibernate OGM 5.2, the version of ORM to use with OGM is 5.2 (use the latest micro version).
```

Comment:
This should be 5.3. Anyway, I would keep this simpler. See other comment for the core section.